### ISSUE TITLE
wrong values calculated

### ISSUE THREAD

**Issue by F6F** [pingswept/pysolar#61]
I installed pysolar 0.7 via pip on raspberry 2B running on arch linux
by trying the examples like given in the documentation I got different results 
```
[alarm@alarmpi ~]$ python3
Python 3.6.3 (default, Oct 30 2017, 14:58:21) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pysolar.solar import *
>>> import datetime
>>> d = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320)
>>> print (d)
2007-02-18 15:13:01.130320
>>> get_altitude(42.206, -71.382, d)
30.91446827139096
```

the value from the documentation is 20.374937135509537 can you help me to bugfix this?
Thank you in Advance

**Comment by rpennel** [comment]
I get something different as well : 

In [54]: d = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320)
In [55]: pys.get_altitude(42.206, -71.382, d)
Out[55]: 24.0476347431011

**Comment by pingswept** [comment]
@F6F My best guess is that Python thinks you're in a different time zone. My next best guess is that there's a difference between Python 3.4 and 3.6, but that's really unlikely.

Here's what happens when I do what you did, but with Python 3.4.3:

```
root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# pip3 install pysolar
Collecting pysolar
  Using cached pysolar-0.7.tar.gz
Building wheels for collected packages: pysolar
  Running setup.py bdist_wheel for pysolar ... done
  Stored in directory: /root/.cache/pip/wheels/c3/d7/df/bbf3a1022c710ccebc3925ef77536d8b88f6640d7891117338
Successfully built pysolar
Installing collected packages: pysolar
Successfully installed pysolar-0.7
root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# python3
Python 3.4.3 (default, Nov 17 2016, 01:08:31)
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pysolar.solar import *
>>> import datetime
>>> d = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320)
>>> print(d)
2007-02-18 15:13:01.130320
>>> get_altitude(42.206, -71.382, d)
20.374937135509537
```

@rpennel If you're also using Python 3.6.x, that would elevate the chances that the Python version was at fault.

**Comment by F6F** [comment]
Am Samstag, 9. Dezember 2017, 20:42:14 CET schrieb Brandon Stafford:
> @F6F My best guess is that Python thinks you're in a different time zone. My
> next best guess is that there's a difference between Python 3.4 and 3.6,
> but that's really unlikely.
> 
> Here's what happens when I do what you did, but with Python 3.4.3:
> 
> ```
> root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# pip3 install
> pysolar Collecting pysolar
>   Using cached pysolar-0.7.tar.gz
> Building wheels for collected packages: pysolar
>   Running setup.py bdist_wheel for pysolar ... done
>   Stored in directory:
> /root/.cache/pip/wheels/c3/d7/df/bbf3a1022c710ccebc3925ef77536d8b88f6640d78
> 91117338 Successfully built pysolar
> Installing collected packages: pysolar
> Successfully installed pysolar-0.7
> root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# python3
> Python 3.4.3 (default, Nov 17 2016, 01:08:31)
> [GCC 4.8.4] on linux
> Type "help", "copyright", "credits" or "license" for more information.
> 
> >>> from pysolar.solar import *
> >>> import datetime
> >>> d = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320)
> >>> print(d)
> 
> 2007-02-18 15:13:01.130320
> 
> >>> get_altitude(42.206, -71.382, d)
> 
> 20.374937135509537
> ```
> 
> @rpennel If you're also using Python 3.6.x, that would elevate the chances
> that the Python version was at fault.

Hi,

which timezone do you have at your side?

Thank you in Advance 
Tobias

**Comment by F6F** [comment]
Am Samstag, 9. Dezember 2017, 20:42:14 CET schrieb Brandon Stafford:
> @F6F My best guess is that Python thinks you're in a different time zone. My
> next best guess is that there's a difference between Python 3.4 and 3.6,
> but that's really unlikely.
> 
> Here's what happens when I do what you did, but with Python 3.4.3:
> 
> ```
> root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# pip3 install
> pysolar Collecting pysolar
>   Using cached pysolar-0.7.tar.gz
> Building wheels for collected packages: pysolar
>   Running setup.py bdist_wheel for pysolar ... done
>   Stored in directory:
> /root/.cache/pip/wheels/c3/d7/df/bbf3a1022c710ccebc3925ef77536d8b88f6640d78
> 91117338 Successfully built pysolar
> Installing collected packages: pysolar
> Successfully installed pysolar-0.7
> root@DESKTOP-ASTOA0M:/mnt/c/Users/Brandon Stafford/Repos# python3
> Python 3.4.3 (default, Nov 17 2016, 01:08:31)
> [GCC 4.8.4] on linux
> Type "help", "copyright", "credits" or "license" for more information.
> 
> >>> from pysolar.solar import *
> >>> import datetime
> >>> d = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320)
> >>> print(d)
> 
> 2007-02-18 15:13:01.130320
> 
> >>> get_altitude(42.206, -71.382, d)
> 
> 20.374937135509537
> ```
> 
> @rpennel If you're also using Python 3.6.x, that would elevate the chances
> that the Python version was at fault.

Hi,
I used: 

from pysolar.solar import *
import datetime
d = datetime.datetime(2007, 2, 18, 15, 13, 1, 
130320,tzinfo=datetime.timezone.utc)
print(d)
print (get_altitude(42.206, -71.382, d))

delivers:
2007-02-18 15:13:01.130320+00:00
30.91446827139096

**Comment by jvhaarst** [comment]
I have the same : 
With 3.4 :
```
Python 3.4.5 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:47:47)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.path.append('/home/jvhaarst/miniconda3/lib/python3.5/site-packages')
>>> from pysolar.solar import *
>>> import datetime
>>> d = datetime.datetime(2007, 2, 18, 15, 13, 1,
... 130320,tzinfo=datetime.timezone.utc)
>>> print(d)
2007-02-18 15:13:01.130320+00:00
>>> print (get_altitude(42.206, -71.382, d))
30.91446827139096
```

With 3.5 : 
```
Python 3.5.4 |Anaconda, Inc.| (default, Oct 27 2017, 18:58:56)
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pysolar.solar import *
>>> import datetime
>>> d = datetime.datetime(2007, 2, 18, 15, 13, 1,
... 130320,tzinfo=datetime.timezone.utc)
>>> print(d)
2007-02-18 15:13:01.130320+00:00
>>> print (get_altitude(42.206, -71.382, d))
30.91446827139096
```

**Comment by jvhaarst** [comment]
But a simple test shows that we are apperently off by 5 hours : 
```
from pysolar.solar import *
import datetime
for delta in range(-12, 12):
    date = datetime.datetime(2007, 2, 18, 15, 13, 1, 130320, tzinfo=datetime.timezone(datetime.timedelta(hours=delta)))
    print(date, get_altitude(42.206, -71.382, date))
```
gives
```
2007-02-18 15:13:01.130320-12:00 -51.48406990867036
2007-02-18 15:13:01.130320-11:00 -42.55595639579684
2007-02-18 15:13:01.130320-10:00 -32.22191968554315
2007-02-18 15:13:01.130320-09:00 -21.2767357056581
2007-02-18 15:13:01.130320-08:00 -10.192189844026334
2007-02-18 15:13:01.130320-07:00 1.0708199531758775
2007-02-18 15:13:01.130320-06:00 11.077682614343283
2007-02-18 15:13:01.130320-05:00 20.374937135509537
2007-02-18 15:13:01.130320-04:00 28.147636965765482
2007-02-18 15:13:01.130320-03:00 33.65236264175932
2007-02-18 15:13:01.130320-02:00 36.13506595521155
2007-02-18 15:13:01.130320-01:00 35.16357901920818
2007-02-18 15:13:01.130320+00:00 30.91446827139096
2007-02-18 15:13:01.130320+01:00 24.0476347431011
2007-02-18 15:13:01.130320+02:00 15.337931181511545
2007-02-18 15:13:01.130320+03:00 5.483583987360827
2007-02-18 15:13:01.130320+04:00 -5.348944497486267
2007-02-18 15:13:01.130320+05:00 -16.395652174808767
2007-02-18 15:13:01.130320+06:00 -27.473457524993275
2007-02-18 15:13:01.130320+07:00 -38.1875370901575
2007-02-18 15:13:01.130320+08:00 -47.926846777643426
2007-02-18 15:13:01.130320+09:00 -55.59424308879547
2007-02-18 15:13:01.130320+10:00 -59.41936522455701
2007-02-18 15:13:01.130320+11:00 -57.933772631313275
```

**Comment by jvhaarst** [comment]
So my guess would be that the examples in the documentation are wrong, but that the code itself is fine.

**Comment by pingswept** [comment]
Ah, I was the person who wrote the examples (and most of the code). When the examples were written, Pysolar operated entirely in UTC, but I was testing and debugging in EST. I bet when we switched to timezone-aware datetimes, the examples were not updated.

Anyway, as always, patches welcome. I think the file that needs updating is here: https://github.com/pingswept/pysolar/blob/master/doc/index.rst

**Comment by jvhaarst** [comment]
Done, PR is here : #63

**Comment by pingswept** [comment]
Fixed in pull request #63. Thanks, Jan!
